api/src/api/index.js

```diff
@@ -3,6 +3,7 @@ const router = express.Router();
 
 router.get('/', (req, res) => res.json('API working!'));
 // Put all routes here
-router.use('/home', require('./home'));
+router.use('/', require('./home'));
```

Comment:
remove `router.use('/', require('./home'))`

Can you also remove `api/src/api/home.js` and `api/src/models/home.js`, and remove any imports for the Home model in `api/src/models/index.js` and `api/test/home.js`?